blog/2025/10-27-androidtv-19/index.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Jellyfin for Android TV 0.19
+authors:
+  - nielsvanvelzen
+slug: androidtv-v0.19.0
+tags: [release, android-tv]
+---
+
+From refreshed user interface elements to playback improvements, there are many changes across the board in the latest update of Jellyfin
+for Android TV.
+
+{/* truncate */}
+
+Welcome back to another release. This time, we've made progress in many aspects of the app. Some more visible than others. This blog post
+will highlight some of the most notable changes. A complete list is available in the
+[GitHub changelog](https://github.com/jellyfin/jellyfin-androidtv/releases/tag/v0.19.0).
+
+## 🔍 Search
+
+![Screenshot of the new search with voice input active](./search.png)
+
+The search functionality of the app was redesigned based on the new style of the app. This new look also introduces voice search! And
```

Comment:
```
 This new look also re-introduces voice search
```
I could have sworn its already a thing?

Reply:
So it's kinda complicated, we had it but due to a bug it would no longer show up on new installs for multiple years. which means a very large group of users never were able to see this. Therefor I'm considering it "new" instead of explaining the issue.